GirafIntegrationTest/tests/test_account_controller.py

```diff
@@ -57,11 +59,16 @@ def test_account_can_get_guardian_id(self):
         Endpoint: GET:/v1/User
         """
         global guardian_id
-        response = get(f'{BASE_URL}v1/User', headers=auth(guardian_token)).json()
-        self.assertTrue(response['success'])
-        self.assertEqual(response['errorKey'], 'NoError')
-        self.assertIsNotNone(response['data']['id'])
-        guardian_id = response['data']['id']
+        response = get(f'{BASE_URL}v1/User/', headers=auth(guardian_token))
+        response_body = response.json()
+        
+        self.assertEqual(response.status_code, HTTPStatus.OK)
+        self.assertIsNotNone(response_body['data'])
+
+        #self.assertTrue(response['success'])
+        #self.assertEqual(response['errorKey'], 'NoError')
+        #self.assertIsNotNone(response['data']['id'])
```

Comment:
The old code should probably be removed. There are a lot of these so I'll only comment on this one.